invenio_app_ils/records/jsonresolver/internal_location.py

```diff
@@ -27,6 +27,7 @@ def location_resolver(internal_loc_pid):
         location_pid = get_field_value(InternalLocation, internal_loc_pid,
                                        Location.pid_field)
         location = Location.get_record_by_pid(location_pid)
+        del location["$schema"]
```

Comment:
Is it needed to remove `$schema` from `location`? For example we remove loan from the item to avoid circular references.